appnav/src/main/kotlin/io/element/android/appnav/LoggedInFlowNode.kt

```diff
@@ -149,6 +150,23 @@ class LoggedInFlowNode @AssistedInject constructor(
             }
         )
         observeSyncStateAndNetworkStatus()
+        observeInvitesLoadingState()
+    }
+
+    private fun observeInvitesLoadingState() {
+        lifecycleScope.launch {
+            repeatOnLifecycle(Lifecycle.State.STARTED) {
+                matrixClient.roomListService.invites.loadingState
+                    .collect { inviteState ->
+                        when (inviteState) {
+                            is RoomList.LoadingState.Loaded -> if (inviteState.numberOfRooms == 0) {
```

Comment:
Question: any idea why it's called `maximumNumberOfRooms` Rust side?

Reply:
I'd say because it doesn't reflect the actual number of rooms we got from SlidingSync ? but not sure